src/shared/ToolTip.jsx

```diff
@@ -0,0 +1,44 @@
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+
+
+{/*
+
+Custom ToolTip component using Bootstrap 2.3.2 and jQuery. For newer Bootstrap release,
+use the InfoTip React component from ./shared
+
+*/}
+
+const Tooltip = ({ title, placement, children }) => {
+  // useEffect hook to initialize the tooltip on component mount & destroy on component unmount
+  useEffect(() => {
+    $('[data-toggle="tooltip"]').tooltip();
```

Comment:
I'd prefer not to add a jQuery dependency if we can avoid it, though I realize that may be tricky with Bootstrap 2. Have you tried the [Tooltip component](https://react-bootstrap.netlify.app/docs/components/overlays/#tooltips) from React Bootstrap? If the CSS classes haven't changed, it may work with Bootstrap 2 as well.